frontend/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
should be changed to sth like `Barcamp Bangkhen 2023`

Reply:
Resolved in [750f15f](https://github.com/BarcampBangkhen/bcbk2023-web/pull/2/commits/750f15feea34aebc5d860d8e1196083b32d620bd)